media-sound/musescore/musescore-2.0.3.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils flag-o-matic
+
+DESCRIPTION="WYSIWYG Music Score Typesetter"
+HOMEPAGE="http://musescore.org/"
+SRC_URI="https://github.com/musescore/MuseScore/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="alsa debug jack mp3 portaudio pulseaudio"
+
+RDEPEND="
+	>=dev-qt/designer-5.3.0:5
```

Comment:
turn all of these into `dev-qt/designer:5` as the version are definitely satisfied